src/libswid.h

```diff
@@ -0,0 +1,80 @@
+#ifndef LIBSWID_H
+#define LIBSWID_H
+
+#include <stddef.h>
+
+#include "swid-common.h"
+
+
+struct SWIDTagIO;
+typedef struct SWIDTagIO * SWIDIOHandle;
+
+
+struct SWIDEntity;
+typedef struct SWIDEntity * SWIDEntityHandle;
+
+
+struct SWIDLink;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this redundant forward declaration; the declaration was already made. ([cpp:S3231](https://sonarcloud.io/organizations/openscap/rules#rule_key=cpp%3AS3231))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=openscap-libswid&pullRequest=9&issues=AWODAzRQdIhI-sZE26p0&open=AWODAzRQdIhI-sZE26p0)

[ISSUE_KEY]: AWODAzRQdIhI-sZE26p0